commands/output/output.go

```diff
@@ -52,6 +53,15 @@ status: %s
 	// Number of lines of context to print for inline comments
 	contextLineCount = 5
 )
+var default_color = map[string]string{
```

Comment:
this should be "defaultColor" rather than "default_color" (i.e. we use mixed case rather than underscores)

Reply:
Sorry, my 8-hours-per-day coding style seems to have become my second nature.